book/src/dev/rfcs/0005-state-updates.md

```diff
@@ -528,30 +528,29 @@ New `non-finalized` blocks are commited as follows:
 
 ### `pub(super) fn queue_and_commit_non_finalized_blocks(&mut self, new: Arc<Block>) -> tokio::sync::broadcast::Receiver<block::Hash>`
 
-1. If a duplicate block exists in the queue:
```

Comment:
We need to revise the RFC to return an error for the older duplicate block.

We won't need a broadcast channel any more.